backend/src/routers/index.py

```diff
@@ -1,11 +1,48 @@
-from fastapi import APIRouter
-
+from fastapi import APIRouter, Depends
+from fastapi.responses import HTMLResponse
+from starlette.responses import RedirectResponse
+from starlette.config import Config
+from starlette.requests import Request
+from authlib.integrations.starlette_client import OAuth
 from backend.src.schemas.index import Success
 
 router = APIRouter()
 
+oauth = OAuth()
+
+CONF_URL = 'https://accounts.google.com/.well-known/openid-configuration'
+oauth.register(
+    name='google',
+    client_id="988417806604-ggnkhrhere0el8b4r3ehko3ncmt8181r.apps.googleusercontent.com",
```

Comment:
@jeffreylam4 this has to be an env var. See my example in the database.py how I retrieve database credentials